terraform/paas/production.env.tfvars

```diff
@@ -0,0 +1,26 @@
+paas_space                = "get-into-teaching-production"
+paas_database_common_name = "school-experience-prod-pg-common-svc"
+paas_redis_1_name         = "school-experience-prod-redis-svc"
+paas_application_name     = "school-experience-app-production"
+paas_internet_hostnames   = []
+application_instances     = 1
```

Comment:
Can you make it 2? The cost is minimal and it's important for HA

Reply:
Yes I have